src/lib/routes/admin-api/api-token.ts

```diff
@@ -312,6 +312,7 @@ export class ApiTokenController extends Controller {
             const token = await this.apiTokenService.createApiToken(
                 createToken,
                 extractUsername(req),
+                extractUserId(req),
```

Comment:
Don't fall back to the system user. I haven't found this being used elsewhere but the default value set to system user is a risk because it's not considering that an admin token should get the admin token user instead